### PR TITLE
Add Dodo Pizza Logo

### PR DESCRIPTION
**Pull request by akaDuality**
Update README.md

### Short description 📝

I would like to add app's logo to the list of companies that are users of Tuist in Readme.

### Checklist ✅

- [x] The code architecture and patterns are consistent with the rest of the codebase.
- [x] The changes have been tested following the [documented guidelines](https://docs.tuist.io/contributors/testing-strategy/).
- [x] The PR includes the label `changelog:added`, `changelog:fixed`, or `changelog:changed`, whenever it should be included in the “Added”, “Fixed” or “Changed” section of the CHANGELOG. Note: when included in the CHANGELOG, the title of the PR will be used as entry, please make sure it is clear and suitable.
- [x] In case the PR introduces changes that affect users, the documentation has been updated.
- [x] In case the PR introduces changes that affect how the cache artifact is generated starting from the `TuistGraph.Target`, the `Constants.cacheVersion` has been updated.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,9 +98,12 @@ The financial sustainability of the project is possible thanks to the ongoing co
   <a href="https://www.olx.com.br" target="_blank">
     <img src="assets/companies/olx.svg" style="width: 60px;" hspace="8"/>
   </a>
+  <a href="https://www.dodopizza.com/" target="_blank">
+    <img src="assets/companies/DodoPizzaLogo.svg" style="width: 50px;" hspace="8/>
+  </a>
```
Comment (danyf90) [inline README.md:103]: Can you please add it after the last one? 🙏

File: README.md [hunk 2]
```diff
@@ -98,9 +98,12 @@ The financial sustainability of the project is possible thanks to the ongoing co
...
   <a href="https://www.justeattakeaway.com/" target="_blank">
     <img src="assets/companies/justeattakeaway.svg" style="width: 110px;" hspace="8"/>
-  </a>
+  </a>
```
Comment (danyf90) [inline README.md:106]:
```suggestion
  </a>
```

**Review by danyf90 — APPROVED** [review]

**Comment by akaDuality** [comment]
It's strange, but when I have added the logo to the end the whole page's layout breaks
Screen Shot 2022-05-26 at 11 26 46
s

**Comment by akaDuality** [comment]
It was missing comma, my bad.

Done, the logo at the last place

**Pull request merged**